kalite/topic_tools/__init__.py

```diff
@@ -491,10 +491,35 @@ def get_assessment_item_data(request, assessment_item_id=None):
     if not assessment_item:
         return None
 
-    # TODO (rtibbles): Enable internationalization for the assessment_items.
+    # Enable internationalization for the assessment_items.
+    try:
+        item_data = json.loads(assessment_item['item_data'])
+        question_content = _(item_data['question']['content'])
+        answerarea_content = item_data['answerArea']['options']['content']
+        if answerarea_content != "":
+            answerarea_content = _(answerarea_content)
+
+        # Loop over contents for multiple hints.
+        # Pickup the wrapped strings and append to hints dict
+        new_hints = []
+        for contents_dict in item_data['hints']:
+            content_value = contents_dict.get('content')
+            converted_content = _(content_value)
+            contents_dict['content'] = converted_content
+            new_hints.append(contents_dict)
+
+        item_data['hints'] = new_hints
+        item_data['question']['content'] = question_content
+        item_data['answerArea']['options']['content'] = answerarea_content
+        # dump the data for a proper json format.
+        assessment_item['item_data'] = json.dumps(item_data)
+
+    except KeyError:
+        logging.error("There is something wrong with the format of the assessment items:%s" % KeyError)
```

Comment:
"something wrong" is a bit vague, can you change it to "Assessment item did not have expected key"? Also, this probably doesn't have the output you expect (`KeyError` will refer to the class `KeyError` and not the instance that you're catching). Plus `KeyError`s are not very verbose.
```python
except KeyError as e:
    logging.error("Assessment item did not have the expected key %s. Assessment item: \n %s" % (e, assessment_item))
```